src/stravalib/model.py

```diff
@@ -357,9 +269,10 @@ class BoundClientEntity(BaseModel):
     bound_client: Optional[Any] = Field(None, exclude=True)
 
 
+# SHould this be typed differently?
```

Comment:
What happens when we remove `Any`? (If that's what you meant?)

Reply:
everything runs fine without Any. so i've removed it. right now i'm passing a string from the dict to the validator. i'll be interested to see when you work on things if there is a better way. but for now i've fixed the typing and my confusing comments (that i think you understood!) as well